README.md

```diff
@@ -1,26 +1,22 @@
 # debugger.html
 
-debugger.html is a hackable debugger for modern times, built from the ground up using [React][react] and [Redux][redux].  It is designed to be approachable, yet powerful.  And it is engineered to be predictable, understandable, and testable.
+debugger.html is a hackable debugger for modern times, built from the ground up using [React] and [Redux].  It is designed to be approachable, yet powerful.  And it is engineered to be predictable, understandable, and testable.
 
-[Mozilla][mozilla] created this debugger for use in the [Firefox][mozilla-firefox] Developer Tools.  And we've purposely created this project in GitHub, using modern toolchains.  We hope to not only to create a great debugger that works with the [Firefox](https://wiki.mozilla.org/Remote_Debugging_Protocol) and [Chrome Debugging Protocol](https://chromedevtools.github.io/debugger-protocol-viewer/1-1/) but development community that can embed this debugger in your own projects with tools like [NPM](http://npmjs.com/).
+[Mozilla] created this debugger for use in the [Firefox] Developer Tools.  And we've purposely created this project in GitHub, using modern toolchains.  We hope to not only to create a great debugger that works with the [Firefox][firefox-rdp] and [Chrome][chrome-rdp] debugging protocols but develop a broader community that wants to create great tools for the web.
 
 <p align="center">
-  <a href="https://devtools-html.slack.com/messages/general/">
-    <img src="https://devtools-html-slack.herokuapp.com/badge.svg">
-  </a>
+  [![slack-badge]][slack]
```

Comment:
Perhaps too far? 😏 

It does ensure we're using the right link everywhere, previously this pointed to https://devtools-html.slack.com/messages/general/ instead of the heroku page https://devtools-html-slack.herokuapp.com/ and I think that's the right way to point people.

Which also reminds me that we should fork the heroku app into `devtools-html` so we can customize it as needed.